utils/src/main/scala/za/co/absa/enceladus/utils/config/ConfigReader.scala

```diff
@@ -60,6 +60,14 @@ class ConfigReader(val config: Config = ConfigFactory.load()) {
     config.getBoolean(path)
   }
 
+  def getIntList(path: String): List[Int] = {
```

Comment:
**Less important:**
The `config.getIntList` is sensitive, that the entry has to be JSON style list of whole number. That makes it impossible to override the value with `=Dkey=value` on command line, as it seems it turns the value into string always.
If not too much code move, would be nice to use the implementation from  https://github.com/AbsaOSS/enceladus/blob/6d1eb17a328fe6d3a4a7fa4b8e4936fad26490d4/utils/src/main/scala/za/co/absa/enceladus/utils/config/ConfigReader.scala#L63 
It even has a good amount of unit tests at https://github.com/AbsaOSS/enceladus/blob/6d1eb17a328fe6d3a4a7fa4b8e4936fad26490d4/utils/src/test/scala/za/co/absa/enceladus/utils/config/ConfigReaderSuite.scala#L116

Reply:
I was thinking about it for some time, but the reason why `getIntList` from the links above works is, I believe, due to `getListFromString` which is a fallback mechanism. Now, in `getListFromString` the whole module `za.co.absa.enceladus.utils.implicits.StringImplicits._` is being loaded, so it's actually a really big piece of code that I would need to copy. I'll still consider it because it might be useful sooner or later anyway - and we already have the code & unit tests, so it's not that I'm gonna actually write a tons of stuff from scratch. Thanks for the comment.